microsoft/testsuites/acc/cvm.py

```diff
@@ -26,6 +26,21 @@ class CVMSuite(TestSuite):
         r"(?P<config_a>(0x[a-z,A-Z,0-9]+)), Group B.(?P<config_b>(0x[a-z,A-Z,0-9]+))"
     )
 
+    # HYPERV_CPUID_ISOLATION_CONFIG.EAX Structure
+    #     UINT32 ParavisorPresent : 1;
+    #     UINT32 Reserved0 : 31;
+    HV_PARAVISOR_PRESENT_MASK = 1
+
+    # HYPERV_CPUID_ISOLATION_CONFIG.EBX Structure
+    #     UINT32 IsolationType : 4;
+    #     UINT32 Reserved11 : 1;
+    #     UINT32 SharedGpaBoundaryActive : 1;
+    #     UINT32 SharedGpaBoundaryBits : 6;
+    #     UINT32 Reserved12 : 20;
+    HV_ISOLATION_TYPE_MASK = 15
+    HV_SHARED_GPA_BOUNDARY_ACTIVE_MASK = 32
+    HV_SHARED_GPA_BOUNDARY_BITS_MASK = 4032
```

Comment:
I have a personal preference for bit masks to be expressed in binary, but others may disagree
``` python
HV_ISOLATION_TYPE_MASK = 0b1111
HV_SHARED_GPA_BOUNDARY_ACTIVE_MASK = 0b100000
HV_SHARED_GPA_BOUNDARY_BITS_MASK = 0b111111000000
```
Similar comment for the asserts below
``` python
 assert_that(
    isolation_config_b & self.HV_SHARED_GPA_BOUNDARY_BITS_MASK
).is_equal_to(0b101110000000)
```

Reply:
I like the binary version more as well. Swapped over to it.

Thanks for the feedback.